cmd/configure/set/set.go

```diff
@@ -0,0 +1,86 @@
+package set
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/cedrata/jira-helper/pkg/config"
+	"github.com/cedrata/jira-helper/pkg/utils"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var (
+	v      *viper.Viper
+	SetCmd *cobra.Command
+)
+
+func init() {
+	SetCmd = &cobra.Command{
+		Use:   "set",
+		Short: "Set profile",
+		Long:  "Set profile in configuration file. If provided profile does not exist, it will be created. If profile is set, the default profile is used. If provided profile already exists, it will be updated.",
+		RunE:  setProfileHandler,
+	}
+
+	_ = SetCmd.Flags().String("name", "", "Name of profile.")
+	_ = SetCmd.Flags().String("host", "", "Host for profile.")
+	_ = SetCmd.Flags().String("token", "", "Token for profile.")
+
+	// Required flags
+	_ = SetCmd.MarkFlagRequired("name")
+	_ = SetCmd.MarkFlagRequired("host")
+	_ = SetCmd.MarkFlagRequired("token")
+}
+
+func setProfileHandler(cmd *cobra.Command, args []string) error {
+	var err error
+	var configPath string
+
+	config.ConfigData = &config.Config{}
+
+	name, _ := cmd.Flags().GetString("name")
+	host, _ := cmd.Flags().GetString("host")
+	token, _ := cmd.Flags().GetString("token")
+
+	v := viper.GetViper()
+
+	if configPath, err = os.UserHomeDir(); err != nil {
+		return err
+	}
+
+	err = config.LoadLocalConfig(configPath, config.DefaultConfigName, v)
+	if _, ok := err.(viper.ConfigFileNotFoundError); !ok && err != nil {
+		return err
+	}
+
+	v.Set(fmt.Sprintf("%v.host", name), host)
```

Comment:
Not the proper behaviour, the file that generates has the wrong format. The default set of values must be under the [defautl] section.

Every new profile name is contained in the square brakets, the host and token fileds are contained in the profile. The used format is ini. As now is not correct how the data is stored

```ini
[default]
host=...
token=...

[profile1]
host=...
token=...
```
and so on

Reply:
I can certainly workaround Viper's functionality and try to update the config file in .ini format, but I wanted to share this with you in case you would consider adopting TOML or YAML file format. 

From the viper docs:

> Q: Why not INI files?

A: Ini files are pretty awful. There’s no standard format, and they are hard to validate. Viper is designed to work with JSON, TOML or YAML files. If someone really wants to add this feature, I’d be happy to merge it. It’s easy to specify which formats your application will permit.

Reply:
It looks like this is a known issue with Viper. Please refer to https://github.com/spf13/viper/issues/1092.

Would you consider adopting TOML or YAML format instead?

Reply:
If the issues does not exists with TOML then i think it could be the way to go, if you can manage to apply this change as well we can proceed with the review of the command

Reply:
I've updated the code to work with TOML format and refactored the "Load Ok" test case.

I believe things work as expected at this point. I am quite busy over the next few weeks so if there are any outstanding changes required I won't be able to work on them. Thanks again for your feedback and letting me pick up this issue!